plugins/action_update_review_status/src/updateReviewsStatuses.ts

```diff
@@ -142,14 +145,23 @@ const updateReviewsStatuses: ActionPluginType = async ({
     // Update review statuses
     for (const review of reviewsToUpdate) {
       const { reviewId, reviewStatus } = review
-      DBConnect.addNewReviewStatusHistory(reviewId, reviewStatus)
+      await DBConnect.addNewReviewStatusHistory(reviewId, reviewStatus)
+    }
+
+    console.log('Resulted reviews assignments to update', reviewAssignmentsToUpdate)
```

Comment:
```suggestion
    console.log('Resulting reviews assignments to update', reviewAssignmentsToUpdate)
```
(And on the other one)

Reply:
Thanks